### PR TITLE
all is done HW-01

### PR DESCRIPTION
**Pull request by ivlytvynets**

### CODE REVIEW

**Review by gigbat — APPROVED** [review]

File: src/main/java/core/basesyntax/Main.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import core.basesyntax.controller.ConsoleHandler;
+
+public class Main {
+
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Main.java]: in all the classes - remove redundant empty line after class declaration

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+
+    private BetDao betDao = new BetDaoImpl();
+
+    public void handle() {
+
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: same

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: what about ArrayIndexOutOfBoundException? let’s catch it here as well

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                System.out.println("Enter, please, valid data");
+            }
+            betDao.add(bet);
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let’s save  the bet if it is not null

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+            System.out.println(bet == null ? null : bet.toString());
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]:
```suggestion
            System.out.println(bet);
```

File: src/main/java/core/basesyntax/model/User.java [hunk 6]
```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.model;
+
+import java.util.List;
+
+public class User {
+
+    private String name;
+    private List<Bet> bets;
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/model/User.java]:
```suggestion
    private int age;
```
let's not bind our entities for now

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**